ui/js/controllers/logviewer.js

```diff
@@ -243,6 +292,42 @@ logViewerApp.controller('LogviewerCtrl', [
 
         /** utility functions **/
 
+        function moveScrollToLineNumber(linenumber, $event) {
+            $scope.currentLineNumber = linenumber;
+
+            $scope.loadMore({}).then(function () {
+                $timeout(function () {
+                    var raw = $('.lv-log-container')[0];
+                    var line = $('.lv-log-line[line="' + linenumber + '"]');
+                    raw.scrollTop += line.offset().top - $('.run-data').outerHeight() -
+                    $('.navbar').outerHeight() - 120;
+                });
+            });
+        }
+
+        function getStepFromLine(linenumber) {
+            var steps = $scope.artifact.step_data.steps;
+            var i;
+            for (i=1; i< steps.length; i++) {
```

Comment:
Let's match the convention used elsewhere in treeherder by consolidating the two lines above and adding suitable operator white space:

```
for (var i = 0; i < steps.length; i++) {
```